src/test/java/seedu/mtracker/console/AddCryptoParserTest.java

```diff
@@ -69,29 +64,17 @@
             + SEPARATOR_SPECIFIER.repeat(2) + FUTURE_DATE
             + SEPARATOR_SPECIFIER + "fooRemarks";
 
-    String formatConsoleInput(String input) {
-        return String.format(input, System.lineSeparator());
-    }
-
-    void simulateConsoleInput(String input) {
-        String formattedInput = formatConsoleInput(input);
-        ByteArrayInputStream inputStreamBytes = new ByteArrayInputStream(formattedInput.getBytes());
-        System.setIn(inputStreamBytes);
-    }
-
-    void checkParameters(AddCryptoParser testCryptoParser, String[] expectedParameters) {
-        for (int i = 0; i < PARAMETER_SIZE; i++) {
-            assertEquals(testCryptoParser.getParameters().get(i), expectedParameters[i]);
-        }
-    }
-
     void testCryptoParameters(String input, String[] expectedParameters) {
         simulateConsoleInput(input);
         AddCryptoParser testCryptoParser = new AddCryptoParser();
         testCryptoParser.initParameters();
         testCryptoParser.getInstrumentParameters();
         checkParameters(testCryptoParser, expectedParameters);
```

Comment:
is there a reason why this is not refactored like the rest of the other test classes?

Reply:
oops, I forgot.